pkg/engine/engine.go

```diff
@@ -154,9 +154,11 @@ func (e *Engine) DoBuild(ctx context.Context, comp *api.Composition, basesrc str
 			return nil, fmt.Errorf("healthcheck and fix errored: %w", err)
 		} else if !rep.FixesSucceeded() {
 			return nil, fmt.Errorf("healthcheck fixes failed; aborting:\n%s", rep)
+		} else if !rep.ChecksSucceeded() {
+			ow.Warnf(aurora.Bold(aurora.Yellow("some healthchecks failed, but continuing")).String())
```

Comment:
nitpick: isn't this printed out as a final result? "continuing" doesn't make sense when this healthcheck is invoked with `testground healthcheck`, i.e. it's a terminal operation in itself.

Reply:
I am not 100% sure this is printed when `testground healthcheck` is called, let me check.

`testground healthcheck` is showing a detailed report of all healthchecks.

Reply:
@raulk actually no need to check. It is obvious this is part of `DoBuild` and `DoRun`.

Reply:
duh, I had overlooked the context